src/tree_sitter_parser.rs

```diff
@@ -473,6 +474,16 @@ pub fn from_language(language: guess::Language) -> TreeSitterConfig {
                 .unwrap(),
             }
         }
+        Perl => {
+            let language = unsafe { tree_sitter_perl() };
+            TreeSitterConfig {
+                name: "Perl",
+                language,
+                atom_nodes: vec!["string"].into_iter().collect(),
```

Comment:
This isn't quite right. If you look at the diff on the sample files, it's not highlighting `'yes'` correctly.

![Screenshot from 2022-04-22 01-05-00](https://user-images.githubusercontent.com/70800/164645524-77b27729-7273-4407-84bb-eab84afa07be.png)

It looks to me like you want `string_single_quoted` and `string_double_quoted` as atoms.

Reply:
Oh, I just copied them from `json`, let me check it out.

Reply:
Fixed.